### DISCUSSION TITLE
Join Pillow Team

### DISCUSSION THREAD

**Discussion by aclark4life** [python-pillow/Pillow#6637] [General]
As mentioned in #4026 by @hugovk, we could use some volunteers to help review PRs. 

And since we are approaching [Year 2 with Tidelift](https://github.com/python-pillow/Pillow/issues/3469), that volunteer position could _potentially_ turn into a paid slot on the Pillow core developer team. If you are interested in volunteering, please tell us your experience with the Pillow project and why you'd like to help.

Thank you all who consider 🎉

**Comment by aclark4life** [comment]
@homm Are you in for [Year 2 with Tidelift](https://github.com/python-pillow/Pillow/issues/3469) i.e. are you still actively developing https://github.com/uploadcare/pillow-simd ? If so, I have two requests:

- **Please let me know by the end of the year** so I can update Tidelift (if we don't hear from you, I'll likely say a sincere "thank you very much for pillow-simd!" and we'll drop to a 4 person team for 2020.)
- **Can we move** https://github.com/uploadcare/pillow-simd **to this organization?** It's fine if we can't for whatever reason, but just wanted to let you know I see the potential for it to become a Pillow-project-proper.

Looks like this was your last Pillow-related activity:

- https://github.com/python-pillow/pillow-perf/commit/674a0871aaa94a89751a94eb036a4cff417b67ef

As far as I can tell from that, https://github.com/uploadcare/pillow-simd is a fork of Pillow which you're using to do performance testing, with results reported to https://github.com/python-pillow/pillow-perf. That's _probably_ OK, but it would be more clear to the outside world I think if the fork of the fork (i.e. https://github.com/uploadcare/pillow-simd) were to be hosted here (and yes, I realize I just stated with sincerity that it would be "more clear if the location of the fork of the fork were to change" … perhaps only slightly more clear 😄 )

**Comment by aclark4life** [comment]
@python-pillow/pillow-team I'm moving this to  ``In Progress`` now, if we don't get anyone by early 2020 let's close this and open a new one toward the end of 2020.

**Comment by homm** [comment]
Hi, @aclark4life and @python-pillow/pillow-team

> are you still actively developing https://github.com/uploadcare/pillow-simd ?

The majority of the work was done before I joined the Lifting program (which was 1 year ago, actually). Right now I'm working on significant improvement of .thumbnail() method in both speed and quality terms, which I hope to finish before 7.0 release (in both Pillow and Pillow-SIMD).

But as before my work is sporadic and based on my primary job's needs. I see how other team members work on Pillow improvement on a regular base and understand what my contribution is very different and may differ from how Tidelifting should look like. So I think that **it is fair to opt-out of the program**.

Despite this, I want to stay Pillow's team member with some weight in discussions and more tolerant code reviews, as I know Pillow's internals very well.

> Can we move https://github.com/uploadcare/pillow-simd to this organization?

This is doable if everyone including users could benefit.

Current problems are:
 * Detained releases (for example, the latest released Pillow-SIMD version is still 6.0.0.post0. Shame on me).
 * Conflicting names. When some package depends on Pillow you need hacks to install Pillow-SIMD.
 * Complicated installation. No binary wheels.
 * Not enough information about Pillow-SIMD and Pillow's performance in general.

I suggest creating a new chapter in the documentation about Pillow's performance where a large part will be about Pillow-SIMD with information about Uploadcare supporting (as no one else still contributed anything in Pillow-SIMD itself). I think a good start is [Pillow-SIMD's README page](https://github.com/uploadcare/pillow-simd#pillow-simd).

I have an idea of how other problems listed above could be solved, but this is part of another topic.

**Comment by aclark4life** [comment]
Thanks @homm , all sounds good to me, thanks for your previous and continued support! If you ever feel like you want to jump in again as "paid volunteer", please let us know.

@kszu can we drop @homm from Tidelift starting Jan 1 2020? Looks like we'll drop to a 4-person team at that point (me, @wiredfool , @hugovk , @radarhere ) and continue to split payments from Tidelift evenly,. Thank you!

**Comment by kszu** [comment]
Yes, we can certainly do that! Thank you for letting us know.

It's been a pleasure working with you @homm ! We'll make our final payment to you in January (accounting for December). Please reach out if you have any other questions!

**Comment by brennaheaps** [comment]
@aclark4life @homm now that the December 2019 payout run is complete I've removed @homm from lifting and adjusted the monthly payment split accordingly. Please let me know if I can help with anything else and thank you!

/cc @kszu

**Comment by hugovk** [comment]
Hi @nulano! Thank you for your continued contributions to Pillow, they're very much appreciated!

We asked a [couple of years ago](https://github.com/python-pillow/Pillow/pull/4084#issuecomment-537105474), but I guess you were too busy at the time. Maybe now's a better time, would you like to join the team? Basically you can carry on pretty much as you are, but can also help out merging PRs and so on.

Cheers!

**Comment by aclark4life** [comment]
@hugovk @nulano And get paid, don't forget getting paid. We're all still splitting our Tidelift revenue evenly (thank you Tidelift!) and we'd all have to vote, of course, but personally I'd have be in favor of a 5-way split. Alternatively, current paid lifters can feel free to say "no" to such an arrangement. This applies to any new addition to the team as far as I'm concerned. New core members may join for some trial period (X number of months) then every so often paid lifters vote on full paid member status for trial members.
